### PR TITLE
Move Read Lock Count Too High to INFO Log

### PR DESCRIPTION
**Pull request by bradyoo**

### CODE REVIEW

File: core/server/common/src/main/java/alluxio/master/StateLockManager.java [hunk 1]
```diff
@@ -138,12 +138,15 @@ public void mastersStartedCallback() {
   public LockResource lockShared() throws InterruptedException {
     if (LOG.isDebugEnabled()) {
       LOG.debug("Thread-{} entered lockShared().", ThreadUtils.getCurrentThreadIdentifier());
+    }
+    if (LOG.isInfoEnabled()) {
       final int readLockCount = mStateLock.getReadLockCount();
-      if (readLockCount > READ_LOCK_COUNT_HIGH) {
+      if (readLockCount > READ_LOCK_COUNT_HIGH && LOG.isInfoEnabled()) {
```
Comment (yuzhu) [inline core/server/common/src/main/java/alluxio/master/StateLockManager.java:144]: nit: `LOG.isInfoEnabled` is always true here right?

Comment (bradyoo) [reply core/server/common/src/main/java/alluxio/master/StateLockManager.java:144]: oops. fixed.

**Review by yuzhu — APPROVED** [review]

**Comment by bradyoo** [comment]
alluxio-bot, merge this please.

**Comment by bradyoo** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please.

**Pull request merged**